frontend/internal-packages/agent/src/chat/workflow/README.md

```diff
@@ -48,22 +55,27 @@ interface WorkflowState {
 
 ## Nodes
 
-1. **validationNode**: Validates inputs (mode, schemaData, projectId) and prepares workflow state for agent execution
-2. **answerGenerationNode**: Selects the appropriate Mastra agent based on mode ('Ask' or 'Build') and generates a response
-3. **finalResponseNode**: Takes the generated answer, formats it as the final response, and updates chat history
+1. **analyzeRequirements**: Organizes and clarifies requirements from user input (performed by pmAgent)
+2. **designSchema**: Designs database schema and executes DDL statements (performed by dbAgent)
+3. **validateSchema**: Verifies use cases and executes DML for testing (performed by qaAgent)
+4. **reviewDeliverables**: Performs final confirmation of requirements and deliverables (performed by pmAgentReview)
+5. **generateArtifacts**: Generates comprehensive artifacts (performed by dbAgentArtifactGen)
+6. **saveToDB**: Saves workflow results to database
```

Comment:
Q: What does this save?
FYI: Saving to supabase can be done during the workflow.

Reply:
Thanks😄, I've updated it accordingly. I've consolidated everything into a node called `finalizeArtifacts`.

![ss 3422](https://github.com/user-attachments/assets/5c92d830-8ada-4ce8-9c20-96540f65e673)


https://github.com/liam-hq/liam/pull/1998/commits/aaa3425d1ec86f97b496c19571f3b70576eae0b4